### PR TITLE
updated ops layers to new architecture and concepts

### PR DESCRIPTION
**Pull request by puja108**
### What this PR does / why we need it

update ops layers to new architecture and concepts as discussed with @pipo02mix

### CODE REVIEW

**Comment by puja108** [comment]
Question was still open (or I forgot what I discussed with Fer) where to move this (and also related docs like the Security doc that mentions Teleport).

I also still have some open questions that I will add comments to here in github

File: src/content/vintage/platform-overview/security/operational-layers/index.md
Comment (puja108) [inline src/content/vintage/platform-overview/security/operational-layers/index.md:46]: where to move cluster access, do we have a new place already, I found Teleport in vintage weird

File: src/content/vintage/platform-overview/security/operational-layers/index.md
Comment (puja108) [inline src/content/vintage/platform-overview/security/operational-layers/index.md:87]: I was not super sure how that looks like in CAPI, do we bind them automatically? do they have access to a kubeconfig? cause the Rest API calls are not available, right?

Comment (puja108) [reply src/content/vintage/platform-overview/security/operational-layers/index.md:87]: I would then also remove the two Rest API related Further readings

File: src/content/vintage/platform-overview/security/operational-layers/index.md
Comment (pipo02mix) [reply src/content/vintage/platform-overview/security/operational-layers/index.md:46]: yeah we did not move that one yet. Maybe as I child of architecture page too

File: src/content/vintage/platform-overview/security/operational-layers/index.md
Comment (puja108) [reply src/content/vintage/platform-overview/security/operational-layers/index.md:87]: ah, so it can still create certs like it used to? with custom org and email as username?

Comment (anvddriesch) [reply src/content/vintage/platform-overview/security/operational-layers/index.md:87]: It's complicated :smiling_face_with_tear:
- We do encourage customers to use our dex app with their IDP of choice on their WCs which works using OIDC in `kubectl gs login` but they need to set up their own RBAC to go with it
- By default `kubectl gs login --workload-cluster...` can be used to create client certificates for workload clusters. In vintage these certs are signed through `Vault` and in CAPI through the CA secret stored for each WC in the org namespace. (so yes, customers need access to the org namespace on the MC to log in)

Comment (anvddriesch) [reply src/content/vintage/platform-overview/security/operational-layers/index.md:87]: Yes. We never got around to introduce any rbac management for wcs sadly. :)

Comment (puja108) [reply src/content/vintage/platform-overview/security/operational-layers/index.md:87]: all good, will try to adjust the wording to reflect

**Review by anvddriesch — APPROVED** [review]

**Comment by puja108** [comment]
@fer do you want to move it to the right place before we merge?

**Comment by pipo02mix** [comment]
Screenshot 2024-12-03 at 14 04 23
yeah please move it next to architecture

**Review by pipo02mix — APPROVED** [review]
LGTM when page is moved to `/overview/architecture` folder

File: src/content/vintage/platform-overview/security/operational-layers/index.md
Comment (puja108) [reply src/content/vintage/platform-overview/security/operational-layers/index.md:46]: should we move that in a separate PR? I think I would not want to extend this one for it

**Comment by puja108** [comment]
@pipo02mix moved PTAL

**Pull request merged**
